src/App/Command/SlackNotifierCommand.php

```diff
@@ -202,39 +202,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     protected function checkStatusNightly(): string
     {
-        $report177x = $this->nightlyBoard->getReport(date('Y-m-d'), '1.7.7.x', 'functional');
-        $reportDevelop = $this->nightlyBoard->getReport(date('Y-m-d'), 'develop', 'functional');
-
-        $has177XPassed = isset($report177x['tests'], $report177x['tests']['passed']);
-        $has177XFailed = isset($report177x['tests'], $report177x['tests']['failed']);
-        $has177XPending = isset($report177x['tests'], $report177x['tests']['pending']);
-        $hasDevelopPassed = isset($reportDevelop['tests'], $reportDevelop['tests']['passed']);
-        $hasDevelopFailed = isset($reportDevelop['tests'], $reportDevelop['tests']['failed']);
-        $hasDevelopPending = isset($reportDevelop['tests'], $reportDevelop['tests']['pending']);
-
-        $status177X = ($has177XFailed && $report177x['tests']['failed'] == 0);
-        $statusDevelop = ($hasDevelopFailed && $reportDevelop['tests']['failed'] == 0);
-
-        $emoji177X = $status177X ? ':greenlight:' : ':redlight:';
-        $emojiDevelop = $statusDevelop ? ':greenlight:' : ':redlight:';
-
         $slackMessage = ':notebook_with_decorative_cover: Nightly Board :notebook_with_decorative_cover:' . PHP_EOL;
-        $slackMessage .= ' - <https://nightly.prestashop.com/report/' . $report177x['id'] . '|' . $emoji177X . ' Report `1.7.7.x`>';
-        $slackMessage .= ' : ';
-        $slackMessage .= $has177XPassed ? ':heavy_check_mark: ' . $report177x['tests']['passed'] : '';
-        $slackMessage .= ($has177XPassed && ($has177XFailed || $has177XPending) ? ' - ' : '');
-        $slackMessage .= $has177XFailed ? ':x: ' . $report177x['tests']['failed'] : '';
-        $slackMessage .= (($has177XPassed || $has177XFailed) && ($has177XPending) ? ' - ' : '');
-        $slackMessage .= $has177XPending ? '⏸️ ' . $report177x['tests']['pending'] : '';
-        $slackMessage .= PHP_EOL;
-        $slackMessage .= ' - <https://nightly.prestashop.com/report/' . $reportDevelop['id'] . '|' . $emojiDevelop . ' Report `develop`>';
-        $slackMessage .= ' : ';
-        $slackMessage .= $hasDevelopPassed ? ':heavy_check_mark: ' . $reportDevelop['tests']['passed'] : '';
-        $slackMessage .= ($hasDevelopPassed && ($hasDevelopFailed || $hasDevelopPending) ? ' - ' : '');
-        $slackMessage .= $hasDevelopFailed ? ':x: ' . $reportDevelop['tests']['failed'] : '';
-        $slackMessage .= (($hasDevelopPassed || $hasDevelopFailed) && ($hasDevelopPending) ? ' - ' : '');
-        $slackMessage .= $hasDevelopPending ? '⏸️ ' . $reportDevelop['tests']['pending'] : '';
-        $slackMessage .= PHP_EOL;
+
+        foreach (['1.7.7.x', '1.7.8.x', 'develop'] as $branch) {
```

Comment:
```suggestion
        foreach (['1.7.7.x', '1.7.8.x', 'develop'] as $branch) {
```
Use const maybe? or configurable option?

Reply:
Fixed